src/file_validator.py

```diff
@@ -421,8 +421,12 @@ def validate_zip_file(archived_files_info, file_path, md5_cache, log):
                 in_archive_manifest = any(row.get(FILE_PATH) == extracted_file_path for row in archived_files_info)
                 if in_archive_manifest:
                     files.append(extracted_file_path)
+                else:
+                    log.error(f"File {extracted_file_path} in zip file {file_path} is not in archive manifest!")
```

Comment:
Please update message to "File {extracted_file_path} found in zip file {file_path} is not included in archive manifest!"